NSelene/SeleneElement.cs

```diff
@@ -330,6 +330,14 @@ ReadOnlyCollection<IWebElement> SeleneContext.FindElements (By by)
             Should(Be.Visible);
             return this.ActualWebElement.FindElements(by);
         }
+
+        public SeleneElement ScrollIntoView()
```

Comment:
I am thinking on adding Js prefix to this kind of methods...
So it will be explicitly visible in tests that we are doing some "magical tricks"... 

what do you think?

Reply:
let's continue the discussion here: https://github.com/yashaka/NSelene/issues/50